examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -1488,6 +1531,9 @@ rtDefineMethod(pxObject, remove);
 rtDefineMethod(pxObject, removeAll);
 rtDefineMethod(pxObject, moveToFront);
 rtDefineMethod(pxObject, moveToBack);
+rtDefineMethod(pxObject, moveToCenter);
+rtDefineMethod(pxObject, moveToCenterH);
+rtDefineMethod(pxObject, moveToCenterV);
```

Comment:
Let's try avoiding adding a new native API unless it's too difficult or impossible to do from javascript.  What is involved in migrating moveToCenter(), moveToCenterH(), and moveToCenterV() from native to javascript?